src/main/java/org/killbill/billing/account/api/AccountUserApi.java

```diff
@@ -159,11 +159,22 @@
     /**
      * Get all the audit entries with history for a given account.
      *
-     * @param accountId   the object id
+     * @param accountId   the account id
      * @param auditLevel audit level (verbosity)
      * @param context    the tenant context
-     * @return all audit entries with history for that object
+     * @return all audit entries with history for an account
      */
     List<AuditLogWithHistory> getAuditLogsWithHistoryForId(UUID accountId, AuditLevel auditLevel, TenantContext context) throws AccountApiException;
 
+    /**
+     * Get all the audit entries with history for a given account email.
+     *
+     * @param accountId   the account id
+     * @param accountEmailId   the account email id
+     * @param auditLevel audit level (verbosity)
+     * @param context    the tenant context
+     * @return all audit entries with history for an account email
+     */
+    List<AuditLogWithHistory> getEmailAuditLogsWithHistoryForId(UUID accountId, UUID accountEmailId, AuditLevel auditLevel, TenantContext context) throws AccountApiException;
```

Comment:
In all of these APIs, I don't think you need the `accountId` as it's already populated in the `context`.